src/app/providers-lite.tsx

```diff
@@ -0,0 +1,35 @@
+'use client'
+
+import { WagmiConfig } from 'wagmi'
+
+import { GTMProvider } from '@elgorditosalsero/react-gtm-hook'
+import { RainbowKitProvider } from '@rainbow-me/rainbowkit'
+
+import { rainbowkitTheme } from '@/lib/styles/theme'
+import { chains, wagmiConfig } from '@/lib/utils/wagmi'
+
+import '@rainbow-me/rainbowkit/styles.css'
+import '@/lib/styles/fonts'
+
+const rainbowKitAppInfo = {
+  appName: 'Index Coop',
+  learnMoreUrl: 'https://indexcoop.com',
+}
+
+// All providers except Chakra
+export function ProvidersLite({ children }: { children: React.ReactNode }) {
```

Comment:
If you check my comment about the main layout file, you probably don't need this. Or did you experience issues when having chakra mixed in?

Reply:
Yeah exactly that, mentioned here: https://github.com/IndexCoop/index-app/pull/990#discussion_r1501333911

(tried a second time today and saw the same issues)